### DISCUSSION TITLE
Porting ImageMagick6 code to 7 and missing transform()

### DISCUSSION THREAD

**Discussion by wbehrens-on-gh** [ImageMagick/ImageMagick#4707] [Help] [answered]
I'm currently trying to port a codebase from imagemagick 6.5.5 to imagemagick 7 and I have a `Magick::Image` class that calls the `.transform(const Geometry &imageGeometry)` method which seems to have been removed at some point. I'm trying to find a replacement for it and was wondering what the most drop-in option would be as I myself am not too familiar with ImageMagick.

**Comment by dlemstra** [comment, marked as answer]
The `.transform` method is no longer available because the internal method is now private instead. You can probably use the `.resize` method instead.
